### PR TITLE
Portfolio: desktop version

### PR DESCRIPTION
**Pull request by jecode93**
## In this milestone I : 

- Use media queries to make the desktop version.
- In order to lay out the elements on the page I use Grid and Flexbox.

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -90,6 +90,10 @@ header a {
   height: var(--hamburger-icon-height);
 }
```
Comment (AbelG101) [inline styles/styles.css:92]: - [ ] It seems like you're forgetting to import the proper font. To ensure consistency with the Figma design, please double-check that all necessary fonts are imported correctly into your project. 
- I've already setup the font you need [here](https://fonts.google.com/specimen/Poppins?query=poppi&preview.text=AWSUGN&preview.text_type=custom) you just have to import it on your HTML or styles file and adjust some of the font-weights in your design so they stay consistent. 😉

File: styles/styles.css
Comment (AbelG101) [inline styles/styles.css:1]: https://github.com/jecode93/portfolio/blob/d17d080ddf04fb4f074c3fabeed410b681ecc58e/styles/styles.css#L64
- [ ] Ouch this is close but as you can see from the Figma design below the background color of the body should be `#f4f5f7` and not `#e5e5e5` so kindly adjust that to stay consistent with the Figma UI. 👌
- Figma design
  image
- Your design:
  firefox_nlo0qFJJwr

**Comment by AbelG101** [comment]
Hello @jecode93 Please restrain from adding features after you've requested a review as this leads to inconsistencies.

**Comment by jecode93** [comment]
> 

Hello @AbelG101 

Sorry for that, I didn't know...
Thank you!

File: styles/styles.css
Comment (AbelG101) [inline styles/styles.css:1]: - [ ] This is close as well but you seem to be missing the implementation of the `:active` or pressed state of the buttons in your design. Paying attention to small details like this is an important trait of a skilled frontend developer. Please make sure to add the `:active` state to ensure a polished user experience.
- Here's the Figma design as reference 👇
  firefox_7KIb3eS8J1

File: styles/styles.css [hunk 2]
```diff
@@ -453,3 +457,126 @@ hr {
...
+    color: #344563;
+    text-align: center;
+  }
+
+  .card {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    height: 496px;
+    align-items: center;
+  }
```
Comment (AbelG101) [inline styles/styles.css]: - [ ] Yet again another small detail seems to be forgotten here as you can see in the Figma design when the project card is hovered over with a mouse there's a border with a `#a7aeff` color and also a slight box shadow. so please make sure to implement those to provide a better UX. ✔
- Figma design:
  firefox_Axe18iHV4e

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @jecode93 👋, 

You're doing a good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
**Your work is impressive! Here are some specific things that stood out to me:**
- The UI is looking fabulous, keep it up! ✔
-  Great file structure, well organized ✔
- You've implemented most of the requirements to perfection ✔
- All linter checks are passing ✔


## Required Changes ♻️
- _Check the comments under the review._
- you just have to make a few changes and you're on your way 🚀.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles/styles.css (hunk 2 above)
Comment (jecode93) [reply styles/styles.css]: @AbelG101 You're right bro I didn't take care of the whole design.
I will do all the changes.

Thank you so much.

**Review by BregornOriginal — APPROVED** [review]
Hi @jecode93 

Your project is complete!  There is nothing else to say other than... it's time to merge it 

Congratulations! 

## Highlights

✅ Linters are passing
✅ Good commit history
✅ Good PR title and description
✅ Your design looks great with the changes applied 💯 

## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.
Cheers and Happy coding!:aplauso::aplauso::aplauso:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, remember to tag me in your question so I can receive the notification.
------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)[.](https://airtable.com/shrQAqnBwek5a0O0s)._)

**Pull request merged**